### PR TITLE
chore: add volunteer kanak22

### PR DESCRIPTION
**Pull request by kanak22**

### CODE REVIEW

File: data/users/Kanak.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Kanak",
+  "bio": "Full Stack developer",
+  "image": "https://avatars.githubusercontent.com/u/62026983?s=400&u=29d76264781ba448718d2c321ef14a849941ac17&v=4",
+  "categories": [
+    {
+      "slug": "programming",
+      "topics": ["C++", "Data structures", "Algorithms","Full Stack"]
```
Comment (codewithvoid) [inline data/users/Kanak.json:8]: here you can drop Full Stack as its mentioned in bio

File: data/users/Kanak.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Kanak",
+  "bio": "Full Stack developer",
+  "image": "https://avatars.githubusercontent.com/u/62026983?s=400&u=29d76264781ba448718d2c321ef14a849941ac17&v=4",
```
Comment (codewithvoid) [inline data/users/Kanak.json:4]: here you can use simpler url https://avatars.githubusercontent.com/u/62026983?v=4

**Comment by codewithvoid** [comment]
Thanks @kanak22 for volunteering. 

Added small comments. PTAL.

File: data/users/Kanak.json (hunk 1 above)
Comment (kanak22) [reply data/users/Kanak.json]: Done

File: data/users/Kanak.json (hunk 2 above)
Comment (kanak22) [reply data/users/Kanak.json]: replaced the url

**Pull request merged**